tests/mock_network/accumulate.rs

```diff
@@ -43,10 +44,11 @@ fn messages_accumulate_with_quorum() {
     let dst = DstLocation::Node(*nodes[closest_elder_index].name()); // The closest node.
     let content = gen_bytes(&mut rng, 8);
 
-    // The BLS scheme will require more than `participants / 3`
+    // The BLS scheme will require more than `participants * 2 / 3`
     // shares in order to construct a full key or signature.
     // The smallest number such that `quorum > threshold`:
-    let threshold = elder_size.saturating_sub(1) / 3;
+    // TODO: make this configurable.
```

Comment:
Is this `TODO` still needed?

Reply:
think so, as currently just caculated from const?

Reply:
Maybe that's enough? We can always change the consts if we want to. Or do you think we might need to change the threshold at runtime? Or have different threshold in different part of the code?

Reply:
I thought we may have different threshold in different part of code.
But I agree this can be adjusted via const easily.
I will remove the TODO

Reply:
Yeah, currently we have 2/3 for parsec votes and 1/3 for message signatures. But I think we eventually want to have 1/2 everywhere. For now however, we will only have 2/3 everywhere.

Reply:
Yes, until parsec is gone then we can get to >50% :+1: